backend/organization/permissions.py

```diff
@@ -241,7 +241,7 @@ def has_permission(self, request, view):
 
         # Detect for AnonymousUser
         if not request.user.is_authenticated:
-            return True
+            return False
```

Comment:
@positiveimpact to be followed up, I actually made this change and then started getting a 401 with this error: https://stackoverflow.com/questions/26906630/django-rest-framework-authentication-credentials-were-not-provided -- changed back in: a7b52d3f

Reply:
Maybe that's some cookie setup problems? We definitely can't return True here, otherwise ever user who isn't authenticated will have admin priviledges :D Maybe it's worth to pair on this issue?

Reply:
So I don't believe this was actually needed, see note in: [c9dddfa](https://github.com/climateconnect/climateconnect/pull/674/commits/c9dddfaf0a4c8a2b37a94e31e20adb11e54e73c2).